src/Compute/Compute/Extension/AzureVMBackup/AzureVMBackupExtensionUtil.cs

```diff
@@ -271,12 +272,24 @@ public void CreateSnapshotForDisks(AzureVMBackupConfig vmConfig, string snapshot
                 ProtectedSettings = privateConfig,
             };
 
-            var vmBackupOperation = virtualMachineExtensionBaseCmdlet.VirtualMachineExtensionClient.CreateOrUpdate(
-                vmConfig.ResourceGroupName,
-                vmConfig.VMName,
-                vmConfig.ExtensionName ?? backupExtensionName,
-                vmExtensionParameters);
-
+            VirtualMachineExtension vmBackupOperation;
+            if (noWait)
+            {
```

Comment:
Same comment - this involves a chain of more than one call, so we should not implement this here.

Reply:
Done